proto/schema/v1/tx.proto

```diff
@@ -15,14 +15,17 @@ service Msg {
 }
 
 message MsgCreateSchema {
-  string creator = 1;
-  string label = 2;
-  string cid = 3;
+  // definition of schema to be persisted
```

Comment:
This will still need the creator for now

Reply:
+1

Reply:
> This will still need the creator for now

yes ...this we will still need the Creator, it got caught in the fire.